spec/support/feature_test_helpers.rb

```diff
@@ -12,10 +12,9 @@ def remove_project_directory
   end
 
   def run_suspenders(arguments = nil)
-    arguments = "--path=#{root_path} #{arguments}"
     run_in_tmp do
       EnvPath.prepend_env_path!(fake_bin_path)
-      command = "#{suspenders_bin} _#{rails_version}_ #{APP_NAME} #{arguments}"
```

Comment:
We the [latest changes](https://github.com/thoughtbot/suspenders/pull/1102), we no longer need to be explicit about the Rails version. It will just work.